lib/pengine/clone.c

```diff
@@ -528,7 +528,7 @@ clone_print(resource_t * rsc, const char *pre_text, long options, void *print_da
     if(rsc->variant == pe_master) {
         enum rsc_role_e role = configured_role(rsc);
 
-        if(role > RSC_ROLE_STOPPED && role < RSC_ROLE_MASTER) {
+        if(role == RSC_ROLE_SLAVE) {
             short_print(list_text, child_text, "Slaves (target-role)", NULL, options, print_data);
```

Comment:
as discussed on IRC, see lib/pengine/utils.c ca 1727, get_target_role(): https://github.com/ClusterLabs/pacemaker/blob/master/lib/pengine/utils.c#L1727
target-role:started, target-role:master, and no target-role at all are all equivalent for m/s resources.
The only target-role preventing promotion, and therefor interesting to be reported, is target-role:slave.